### PR TITLE
chore: fix licencing to allow for open source community support

### LINKED ISSUE #6: Licence violates github TOS
* No Distribution of Modified Versions: You may not distribute modified versions of the software, whether in source or binary form.
    * No Forking: You may not create, maintain, or distribute a forked version of the software.
    * Official Distribution: Only the maintainers of the official repository are allowed to distribute the software and its modifications.

https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#5-license-grant-to-other-users

> By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of Content from your repositories in repositories they control).

### PR DESCRIPTION
**Pull request by niceEli**
resolves #6 

This PR would allow for the winamp community to be able to do whatever with winamp, now i know that all you think is numbers and sales and junk, HOWEVER winamp 100% will die from licensing issues, you should prob merge this. whoever is maintaining you probably don't understand licensing, BUT you should merge this for the sake of winamp, preservation, and the community to forever use this app. also i will finally state that the current license is technically void because of it being incorrect, stating its copyleft, automatically applying this GPL license to it as its considered copyleft.

### **A COPYLEFT Licence Is Part Of Open Source**

### CODE REVIEW

**Comment by niceEli** [comment]
TLDR for maintainers, technically my pr is already "merged" because of your license stating copyleft.

**Comment by GarlandKey** [comment]
AFAIK, their license isn't enforceable because it violates the GitHub ToS.

**Comment by DimaLeon2000** [comment]
Why not the more permissive MIT license?

**Comment by Lastorder-DC** [comment]
> Why not the more permissive MIT license?

MIT license is no longer possible since winamp already used GPL code at their source code.(as I know) @DimaLeon2000

**Comment by omarsaad98** [comment]
Please fix the git change to be a move instead of a delete + create. you can do this with [`git mv`](https://git-scm.com/docs/git-mv)

**Comment by niceEli** [comment]
> Please fix the git change to be a move instead of a delete + create. you can do this with [`git mv`](https://git-scm.com/docs/git-mv)
why would it be needed to use a move, its best to fully abandon the (most likely ai generated) trash licence, instead using a real copyleft licence.

**Review by 1vanK — COMMENTED** [review]
approved
